assets/dev/scss/frontend/widgets/rating.scss

```diff
@@ -4,6 +4,9 @@
 	--e-rating-star-color: #{$gray-mouse};
```

Comment:
We need to change the default variables:

`.elementor-widget-rating {`
	`--e-rating-gap: 0px;`
	`--e-rating-star-font-size: 16px;`
	`--e-rating-star-color: #{$gray-mouse};`
	`--e-rating-star-marked-color: #{$orange};`
	`--e-rating-star-marked-width: 100%;`
  	`--e-rating-justify-content: flex-start;`
`}'

`.e-rating {`
	display: flex;
  	justify-content: var(--e-rating-justify-content);

Reply:
Done